rfcs/0050-roadmaps.md

```diff
@@ -0,0 +1,320 @@
+considered the "owners" of a roadmap, and generally no rigid process is defined around modifying the
+items in a roadmap.
+
+Roadmaps are defined using [GitHub
+Projects](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects)
+and can include GitHub Issues, Pull Requests, and simple note cards. Maintainers should strive to
+place mainly **GitHub Issues** in Roadmaps to make it possible for the community to learn more about
+ongoing work and reduce triage burden.
+
+Each item on a roadmap is intended to track one of these community processes:
+
+- **pre-RFCs** serve as a way to begin discussions on planned work at the earliest stage of
+  maturity. **pre-RFCs** are typically posted in the [TVM Discuss
+  forums](http://discuss.tvm.apache.org) in order to solicit TVM Community feedback. For an example
+  of a **pre-RFC**, see the screenshot of Andrew R's proposal to *Convert RST Docs to Markdown*
+  below.
+
+    **pre-RFCs** can be tracked on a Roadmap by preemptively creating a GitHub Task-tracking Issue
+    in [tvm-rfcs](https://github.com/apache/tvm-rfcs).
+    ![image|690x460](./assets/0050/roadmap-item-pre-rfc.png)
+
+- **RFCs** serve as a way to share feature proposals with the TVM Community. The
+  [tvm-rfcs](https://github.com/apache/tvm-rfcs) repo is used for the creation and acceptance of
+  **RFCs**. For some examples of **RFCs**, see the screenshot of open pull requests in
+  [tvm-rfcs](https://github.com/apache/tvm-rfcs) below.
+
+    Open **RFCs** can be directly linked into any roadmap. Once an **RFC** is accepted, please use
+    the **Github Task-Tracking** process to track **RFC** Execution.
+    ![image|690x357](./assets/0050/roadmap-item-rfc.png)
+
+- **Github Task-Tracking Issues** are used in [tvm](https://github.com/apache/tvm) to share the
+  progress of accepted RFCs over time. For an example of a **Github Task-Tracking Issue**, see the
+  screenshot of Andrew L's RFC to *Add Mixed-Precision Support to TVM* below.
+
+    These task-tracking issues can be directly linked into any roadmap.
+    ![image|663x499](./assets/0050/roadmap-item-gh-task-tracking-issue.png)
+
+- **Bugfixes** are actionable GitHub Issues not necessarily connected to a Task-Tracking
+  Issue. Generally, they require only 1 PR and the work is clearly specified in the issue. For an
+  example of a **bugfix** Github Issue, see the screenshot of a flaky CI test report below.
+
+    You can directly link **bugfixes** in a roadmap by adding their corresponding GitHub Issues as a
+    card. Generally, it’s the maintainer’s responsibility to combine related **bugfixes** in order
+    to keep the roadmap concise. For example, if you have 10 flaky tests in CI, it might make sense
+    to have a global tracking issue for flaky CI tests which points to each individual bug.
+     ![image|690x368](./assets/0050/roadmap-item-bugfix.png)
+
```

Comment:
There is a category of mid-sized issues missing here that people generally don't file right now, but there is a lot of work not directly related to implementing an RFC or a bug fix that goes on (i.e. bigger than a bug fix but smaller than an RFC). I don't think there's any harm in making these into issues, even if they're not actionable/owned right away or if the ideas are less well formed (maybe the issue is just so a specific developer doesn't forget about something).

This would be nice with a wider application of the triage role since devs writing issues could assign it to themselves or mark it as triaged so it doesn't fill the queue.

Reply:
This is a great callout. I agree that a roadmap's maintainers should have the ability to track these types of issues on the roadmap.

Perhaps this falls under Github Task-Tracking, and we should just change the wording so that we're not boxing that particular category in as "accepted RFCs only" - what are your thoughts @driazati @areusch?

Reply:
I think that'd be good @denise-k, we've done this in the past to track future work arising from a PR review (for example: https://github.com/apache/tvm/issues/8792) - this should serve as a reminder and incentive to follow up :smile_cat:

Reply:
Thanks! I've added a mini-section here to address this.